src/client.cpp

```diff
@@ -1078,9 +1082,16 @@ void Client<IoType, SwitchDataIntegrity, SwitchActivityInfo, SwitchCycleDuration
                 // Disarm timer
                 timer.it_value.tv_sec = 0;
                 timer.it_value.tv_usec = 0;
+
+#ifdef WIN32
+                if (SetTimer(NULL, 0, 0, NULL)) {
+                    log_err("ERROR: SetTimer() failed when disarming");
+                }
+#else
                 if (setitimer(ITIMER_REAL, &timer, NULL)) {
                     log_err("ERROR: setitimer() failed when disarming");
                 }
+#endif
```

Comment:
please consider moving os dependencies into `os_abstarct.[h|cpp]` if possible.
For example:
```
int os_settimer(timer)
{
#ifdef WIN32
#else
#endif
}
```
Pay attention that Linux and Windows return different values but it can be unified to linux style.